test_podcastparser.py

```diff
@@ -50,3 +53,16 @@ def test_parse_rss(rss_filename):
 
     for rss_filename in glob.glob(os.path.join('tests', 'data', '*.rss')):
         yield test_parse_rss, rss_filename
+
+def test_invalid_roots():
+    def test_fail_parse(feed):
+        with assert_raises(podcastparser.FeedParseError):
+            podcastparser.parse('file://example.com/feed.xml', StringIO(feed))
+
+    feeds = [
+        '<foo><bar/></foo>',
+        '<foo xmlns="http://example.com/foo.xml"><bar/></foo>',
+        '<baz:foo xmlns:baz="http://example.com/baz.xml"><baz:bar/></baz:foo>',
```

Comment:
I considered that testing against `<foo>` is essentially as good as testing against `<html>`, since the name itself doesn't matter to the logic. I used `foo` so that it would be as unlikely as possible that it might be used for some future feed standard.

Should we simply change the plain `foo` test to `html`? Or were you thinking that it should also test against a more complete HTML document, including XML header, DOCTYPE, etc.?